docs/api/endpoints/users.md

```
@@ -8,6 +8,25 @@ credential — in [Authentication](../authentication.md).
 
 Returns the authenticated user's profile.
 
+## Get Current Usage
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)